### PR TITLE
Refactored server code to serve static files from Express server

### PR DESCRIPTION
**Pull request by chrystalzou**

### CODE REVIEW

File: server/src/index.js [hunk 1]
```diff
@@ -3,10 +3,14 @@ const bodyParser = require('body-parser');
...
 const express = require('express');
 const app = express();
+const path = require('path');
 
 app.use(bodyParser.json());
 
-app.get('/', (req, res) => res.send('CavaTable reviews module server is working!'));
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname + '../../../index.html'));
+    res.sendFile(path.join(__dirname + '../../../index.html'));
```
Comment (asownder95) [inline server/src/index.js]: Is there a reason for the duplicate lines here?

**Review by asownder95 — COMMENTED** [review]
Looks good! Left a comment at a potential duplicate line.

File: server/src/index.js (hunk 1 above)
Comment (chrystalzou) [reply server/src/index.js]: Removed it! Nice catch.

**Review by asownder95 — APPROVED** [review]
Cool!

**Pull request merged**